### PR TITLE
Add dark mode to dashlane business logo

### PR DESCRIPTION
**Pull request by Alexandre-Herve**
Allow passing one color for "dashlane" and one color for "business".
Also fixing typos and style from my previous PRs

### CODE REVIEW

File: src/components/business-logo/BusinessLogo.tsx [hunk 1]
```diff
@@ -2,79 +2,108 @@ import * as React from 'react';
...
-    </React.Fragment>
-  </SvgWrapper>
-);
+export const BusinessLogo: React.FunctionComponent<
+  BusinessLogoProps
+> = props => {
+  const { dashlaneColor, businessColor, ...otherProps } = props;
+
+  const dashlaneFill = dashlaneColor || '#0E6476';
+  const businessFill = businessColor || '#0E353D';
```
Comment (sericaia) [inline src/components/business-logo/BusinessLogo.tsx:33]: I believe you can import the colors instead of adding #0E6476 and #0E353D

Comment (Alexandre-Herve) [reply src/components/business-logo/BusinessLogo.tsx:33]: yes yes thanks I'm doing that

Comment (sericaia) [reply src/components/business-logo/BusinessLogo.tsx:33]: other than that looks good, thanks for doing it :)

**Comment by Alexandre-Herve** [comment]
Ready for review now

File: src/components/business-logo/BusinessLogo.tsx [hunk 2]
```diff
@@ -1,80 +1,101 @@
...
 };
 
+interface CustomProps {
+  /**
+   * Sets the logo for dark mode: "Dashlane" becomes white
+   * instead of dashGreen00.
+   * @default false
+   */
+  darkMode?: boolean;
+}
```
Comment (oclero) [inline src/components/business-logo/BusinessLogo.tsx:18]: Note that this should be a temporary method to change theme, before we actually implement theming in the library.

Comment (Alexandre-Herve) [reply src/components/business-logo/BusinessLogo.tsx:18]: Okay, I didn't know that it was on tracks!

**Comment by sericaia** [comment]
you can edit the PR description since we can only change one color now ;p

**Comment by Alexandre-Herve** [comment]
Ah damn, it's impossible to have it perfect haha

**Pull request merged**